libs/unity/library/Runtime/Scripts/Media/MediaLine.cs

```diff
@@ -332,11 +363,20 @@ internal void OnUnpaired(MediaTrack track)
             // mode (per the WebRTC spec). So the SetDirection(sendonly) triggers the TrackRemoved
             // event, but the pairing was never done because SetDirection() is called before
             // the received is updated.
-            if (_isReceiverPaired)
+
+            // Callbacks must be called on the main thread.
+            _peer.InvokeOnAppThread(() =>
```

Comment:
I'd move the 2 asserts related to `Transceiver` above in that app thread delegate (can't leave comment on them, scroll up), that ensures we're not accessing them concurrently, no?

Reply:
I replaced them with an assert on the track which is more relevant.